flake.nix

```
@@ -19,6 +19,9 @@
                     projectPath = ./.;
                     packages = with pkgs; [
                         # Native dependencies, e.g. imagemagick
+
+                        # Used on local development to catch outgoing emails
+                        mailhog
```

Maybe we can have this here only as a comment? Otherwise it will increase the time to first install of IHP, as it needs to download even more packages then by default

```suggestion
                        # mailhog
```